test/example/v3/definitions/openapi.json

```diff
@@ -0,0 +1,91 @@
+{
```

Comment:
That's our bad. Indeed the generated openapi.json file shows error in the swagger editor. But when we render it with the latest swagger-ui component, it seemed to be correctly rendered without error...
Anyway, I will fix the example so that it is accepted by swagger editor and compliant with openapi 3.x spec.